policyengine_us/parameters/gov/states/ct/tax:income/main/head_of_household.yaml

```
@@ -0,0 +1,40 @@
+description: CT head of household income tax rate
+metadata:
+ threshold_unit: currency_USD
+ rate_unit: /1
+ threshold_period: year
+ type: marginal_rate
+ label: CT head of household income tax rate
```

Also ensure two spaces in other lines, and spell out the state in labels
```suggestion
  label: Connecticut head of household income tax rate
```